src/Helpers.py

```diff
@@ -1,8 +1,9 @@
+import csv
 import os
 import pkgutil
 import json
 
-from BaseClasses import MultiWorld, Item, Location

```

Comment:
ManualLocation comes from Location and is still used below, so we need to add this back.

Reply:
It's not used at all in this file

Reply:
Oh, you're right. Misread, carry on :smiley: